resources/docker-build_p2p/ssh.rc

```diff
@@ -4,8 +4,11 @@
 ### the IP or DN of the p2p server
 p2p_server=86.119.39.89
 
+### the port of the http service in the p2p server
+http_port=800
```

Comment:
I'm a bit confused - why did you use variable "p2p_port" without any initial value but this one with 800? 
I haven't read into any documentation or so on and forgive me if it is already there, but we should add it :)